ReleaseNotes.md

```diff
@@ -7,6 +7,33 @@ samples distributed with the code.  The samples demonstrate the usage of compres
 public models and datasets for three different use cases: Image Classification, Object Detection,
 and Semantic Segmentation.
 
+## New in Release 2.0.2
+Target version updates:
+- Relax TensorFlow version requirements to 2.4.x
+
+## New in Release 2.0.1
+Target version updates:
+- Bump target framework versions to PyTorch 1.9.1 and TensorFlow 2.4.3
+- Increased target HuggingFace transformers version for the integration patch to 4.9.1
+
+Bugfixes:
+- Fixed statistic collection for the algo mixing scenario
+- Increased pruning algorithm robustness in cases of a disconnected NNCF graph
+- Fixed the fatality of NNCF graph PNG rendering failures
```

Comment:
whether `fatality`, or `failures` seems to be redundant
```suggestion
- Fixed the fatality of NNCF graph PNG rendering
```

Reply:
I'll have to stand by my own wording here - currently the failures may still occur, it's just that they won't be fatal to the entire script run.

Reply:
ok, got it